boards/nucleo-f767zi/include/periph_conf.h

```diff
@@ -118,7 +118,7 @@ static const uart_conf_t uart_config[] = {
 static const spi_conf_t spi_config[] = {
     {
         .dev      = SPI1,
-        .mosi_pin = GPIO_PIN(PORT_A, 7),
+        .mosi_pin = GPIO_PIN(PORT_B, 5),
```

Comment:
```suggestion
#ifndef MODULE_PERIPH_ETH
        /* PA7 is also used by the Ethernet device, only use it */
        /* if the Ethernet features is not used to keep the */
        /* Arduino pinout compatibility */
        .mosi_pin = GPIO_PIN(PORT_A, 7),
#else
        .mosi_pin = GPIO_PIN(PORT_B, 5),
#endif
```

The other solution is to add:
```mk
FEATURES_CONFLICT += periph_eth:periph_spi
FEATURES_CONFLICT_MSG += "The Ethernet device and SPI configured on Arduino pinout share the same pin."
```
in the board `Makefile.features` but it's more static => one cannot use both SPI and Eth in the same application unless commenting out the lines in the Makefile and reassigning the MOSI pin in the config.

Personally, I prefer the first solution.

Reply:
+1 for the first solution

Reply:
Agreed. @kaspar030: You suggested to go consistently for the pins marked as `SPI_B` on the silkscreen instead of only changing one pin. However, PB3 (SCK in SPI_B) is also marked to have a pin conflict:

![CN7 and CN10 connectors of the Nucleo](https://os.mbed.com/media/uploads/jeromecoutant/nucleo_f767zi_zio_right_2020_3_30.png)

![Legend](https://os.mbed.com/media/uploads/bcostm/pinout_legend_2017-06-28-2.png)

(Pictures published on [os.mbed.com](https://os.mbed.com/platforms/ST-Nucleo-F767ZI/))